app/forms/waste_carriers_engine/edit_form.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module WasteCarriersEngine
+  class EditForm < BaseForm
+    delegate :reg_identifier, to: :transient_registration
```

Comment:
this is already defined in the BaseForm :) 
```suggestion
```

Reply:
Oops! Fixed: 90eb916